communication/slack-config/channels.yaml

```diff
@@ -3,6 +3,7 @@
 
 channels:
   - name: airflow-operator
+  - name: alibaba-cloud
```

Comment:
since this is a subproject in sig-cloudprovider, we should follow the naming prefix that other providers are using ( like `provider-alibaba`) and move it to [sig-cloud-provider/config.yaml](https://github.com/kubernetes/community/blob/master/communication/slack-config/sig-cloud-provider/config.yaml).

ref: https://github.com/kubernetes/community/pull/4193
/cc @cheftako

Reply:
+1, good catch